bench/workgen/workgen.swig

```diff
@@ -105,11 +105,10 @@ import numbers
 %extend workgen::classname {
 %pythoncode %{
     def __setattr__(self, attr, val):
-        if getattr(self, attr) == None:
+        if attr != 'this' and getattr(self, attr) == None:
             raise AttributeError("'" + #classname +
               "' object has no attribute '" + attr + "'")
-        f = _workgen.__dict__[#classname + '_' + attr + '_set']
-        f(self, val)
+        object.__setattr__(self, attr, val)
```

Comment:
Is this change for code refactoring?

Reply:
@lukech, this change is a clean up or clarification.  SWIG generates a function named something like `_workgen.ThreadOptions_throttle_set` to set the `throttle` attribute of `ThreadOptions`.  But that function always just sets the attribute.  I hadn't realized that previously, and once I did, I made this change.  Rather than look up the function by name and calling it, I just do what it does: set the attribute.

Reply:
Got it, thanks!